pages/home.md

```diff
@@ -19,11 +19,6 @@ permalink: /
       applications and services to use with it.
       </p>
```

Comment:
Maybe needs a closer saying the vision os Solid is a rich world og great power and great fun for devs but it is early days, not ready for end users, but certainly as a dev do join the solid/chat group and hep us bild this new world. If you are a user, watch this space. If  are an organization lookin to get involved then do connect with other companies and agencies which driving this.  Do come to Solid World.  or something

Reply:
I know you made this comment on the homepage but I ended up reflecting it on the about page in a new section called "How can i get involved" because it seemed to fit more naturally there